test/specs/repo/queryservice.ts

```diff
@@ -277,9 +277,9 @@ describe( 'QueryService', function () {
 	} );
 
 	it( 'Should show results from a page in allowlist.txt', async function () {
-		// We don't currently have a way for the example suite to pass tests with breaking changes
+		// We don't currently have a way for the Deployment Kit to pass tests with breaking changes
```

Comment:
just an honest question to make sure: is shortening to "Deployment Kit" alright or should we stick to "WBS Deployment Kit" for coherence? Can there be any confusion at all?

Reply:
Good point. Deployment Kit IS very long. Maybe we can have one more talk about the name. You are not the only one questioning the excessive length ^^

Reply:
follow up in https://phabricator.wikimedia.org/T364957